statusboard/src/components/ProjectsTable/ProjectsTable.scss

```diff
@@ -16,11 +17,12 @@
     width: 100%;
     border-spacing: 0;
     border-radius: inherit;
+    table-layout:fixed;
```

Comment:
I didn't know about `table-layout: fixed`, but it seems good for this purpose! Even more so if we end up adding better paging/infinite scrolling.